### DISCUSSION TITLE
기등록송장 오류

### DISCUSSION THREAD

**Discussion by SungjungC** [commerce-api-naver/commerce-solutions#60] [묻고 답하기]
안녕하세요.

해당 api 호출 시 기등록송장 오류 발생하였습니다.

확인해보았으나 같은 송장번호로 등록된 주문이 조회가 되지 않는데 확인 가능하실까요?

다음은 api 호출 전문입니다.

https://api.commerce.naver.com/partner/v1/pay-order/seller/product-orders/dispatch
POST
{
    "dispatchProductOrders": [
        {
            "productOrderId": "2024111241029611",
            "deliveryMethod": "DELIVERY",
            "deliveryCompanyCode": "CJGLS",
            "trackingNumber": "540334175472",
            "dispatchDate": "2024-11-15T10:29:39.646"
        }
    ]
}

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

호출 경로상으로는 커머스솔루션임을 확인할 수 없습니다.
이 공간은 커머스솔루션 개발사를 위한 기술지원 공간인데요, 혹시 등록하신 솔루션의 이름이나 솔루션 ID 혹은 개발사 이름을 알 수 있을까요?

**Reply by SungjungC** [reply]
@commerce-api-naver 
볼드나인 입니다!

**Reply by commerce-api-naver** [reply]
@SungjungC 님, 커머스API 이민석입니다.

API대행사는 대행사 전용 기술지원 공간으로 문의해주시기 바랍니다.
- https://github.com/commerce-api-naver/api-agency
